### ISSUE TITLE
Long book titles are unreadably small 

### ISSUE THREAD

**Issue by xbao** [PoE-unofficial/Pillars-of-Eternity-bugtracker#7]
Game version: Steam 1.03
System info: Linux Mint 17.1
GPU: 720M
Driver: NVIDIA binary driver - version 331.38
Screen: 1920x1080, 15 inch

Books with long names have their titles displayed in a barely readable font size on a 15" screen. Screenshots probably don't show the issue, but here are a couple anyway

![screenshot from 2015-04-04 09 32 28](https://cloud.githubusercontent.com/assets/8990412/7056815/8cb093f0-de92-11e4-9e7f-37102f1e8b0c.png)

![screenshot from 2015-04-04 09 32 34](https://cloud.githubusercontent.com/assets/8990412/7056836/b2aa7b7a-de92-11e4-913d-4845c3b0ea81.png)

**Comment by shmerl** [comment]
Also, at least for now, please provide a link to the corresponding forum thread, since developers so far didn't confirm that they watch this bug tracker.

**Comment by xbao** [comment]
Done

**Comment by dscharrer** [comment]
Interestingly they're much more readable here at 1920×1200 with font size set to 85%
![](http://i.imgur.com/D4auOEf.jpg)

**Comment by ushass** [comment]
You are right, besides to the length of the title, it's related to the chosen size of the text in the graphical settings. So this may happen for shorter texts when using bigger font size...

I guess they try to fit 2 lines for the title?

For example, when using bigger font, Kana's (Chanter's) song names looks like:
![Chants](http://s14.postimg.org/3qdfxwdup/Chants.jpg)

[1920x1080]
Again using font size 130%, following lore books are probably problematic:
lore_book_backer_rpg_codex.unity3d
lore_book_dyrwoodan_farce_1.unity3d
lore_book_dyrwoodan_farce_2.unity3d
lore_book_eir_glanfath_2.unity3d
lore_book_god_ondra.unity3d
lore_book_knights_of_the_crucible_1.unity3d
lore_book_knights_of_the_crucible_2.unity3d
lore_book_readceran_morality_play_1.unity3d
lore_book_readceran_morality_play_2.unity3d
lore_book_the_aedyr_empire_2.unity3d
lore_book_the_blacksmith_knights.unity3d
lore_book_the_dyrwood_3.unity3d
lore_book_the_dyrwood_5.unity3d
lore_book_the_dyrwood_6.unity3d
lore_book_vailian_tragedy.unity3d
lore_book_woedican_holy_text.unity3d

Books below show only smaller text in tooltips (on 130% font):
lore_book_the_dyrwood_1.unity3d
lore_book_the_dyrwood_2.unity3d
lore_book_monsters_deadfire_archipelago.unity3d
lore_book_glanfathan_poetry.unity3d
lore_book_discoveries_of_padgram.unity3d
lore_book_the_aedyr_empire_1.unity3d
lore_book_god_abydon.unity3d
lore_book_the_dyrwood_4.unity3d

**Comment by curi** [comment]
this is a mac issue too, not just linux.
